CRM/Core/Form/Task.php

```diff
@@ -167,7 +167,7 @@ public function setContactIDs() {
   }
 
   /**
-   * Simple shell that derived classes can call to add buttons to
+   * Simple shell that derived classes can call to add buttons to.
    * the form with a customized title for the main Submit
```

Comment:
I know it's not very good English, but the comment here appears to be (attempting) a complete sentence:

> Simple shell that derived classes can call to add buttons to the form

Makes even less sense when you add the full stop

> Simple shell that derived classes can call to add buttons to. the form

Reply:
How about:

    /**
     * Adds a pair of buttons to the form.

Reply:
@colemanw fixed